data_analysis/static_gtfs_analysis.py

```diff
@@ -239,90 +257,36 @@ def group_trips(trip_summary: pd.DataFrame, groupby_vars: list, hourly: bool):
         .reset_index()
     )
 
-    if hourly:
-        summary.rename(
-            columns={
-                "arrival_hour": "hour",
-                "trip_id": "trip_count",
-                "raw_date": "date"},
-            inplace=True,
-        )
-
-    else:
-        summary.rename(
-            columns={
-                "trip_id": "trip_count",
-                "raw_date": "date"},
-            inplace=True
-        )
+    summary.rename(
+        columns={
+            "trip_id": "trip_count",
+            "raw_date": "date"},
+        inplace=True
+    )
     summary.date = summary.date.dt.date
     return summary
 
 
-def summarize_and_save(trip_summary: pd.DataFrame,
-                       hourly: bool = True,
-                       save: bool = True) -> pd.DataFrame:
-    """Get trips by hour, date, and route
+def summarize_date_rt(trip_summary: pd.DataFrame) -> pd.DataFrame:
+    """Summarize trips by date and route
 
     Args:
         trip_summary (pd.DataFrame): a summary of trips with one row per date.
             Output of the make_trip_summary function.
-        hourly: (bool, optional): whether to aggregate by arrival_hour.
-            Defaults to True.
-        save (bool, optional): whether to save to S3 bucket. Defaults to True.
 
     Returns:
-        pd.DataFrame: A DataFrame grouped by date, hour, and route
+        pd.DataFrame: A DataFrame grouped by date and route
     """
     trip_summary = trip_summary.copy()
     groupby_vars = ["raw_date", "route_id"]
 
-    if hourly:
-        groupby_vars.append("arrival_hour")
-        # group to get trips by hour by date by route
-        route_daily_summary = group_trips(
-            trip_summary,
-            groupby_vars=groupby_vars,
-            hourly=hourly
-        )
-
-        if save:
-            route_daily_summary.to_csv(
-                f"s3://{BUCKET}/schedule_summaries/route_level"
-                f"/schedule_route_daily_hourly_summary_v{VERSION_ID}.csv",
-                index=False,
-            )
-
-        # group to get trips by hour by date by route by *direction*
-        groupby_vars.append("direction")
-        route_dir_daily_summary = group_trips(
-            trip_summary,
-            groupby_vars=groupby_vars,
-            hourly=hourly
-        )
-
-        if save:
-            route_dir_daily_summary.to_csv(
-                f"s3://{BUCKET}/schedule_summaries/route_dir_level/"
```

Comment:
Is it worth it to start saving the daily equivalent of these files to s3? Something like `f"s3://{BUCKET}/schedule_summaries/route_dir_level/schedule_route_dir_daily_summary_v{VERSION_ID}.csv"`?

Reply:
Yes, I think we should, but I can tackle that (or I can look into giving you write access to the S3 bucket). Either way I'd be inclined to merge this PR as-is and then handle.